.github/workflows/benchmarks.yml

```diff
@@ -172,13 +172,13 @@ jobs:
             --zkvyper './target-zkvyper/release/zkvyper' \
             --path=${{ inputs.compiler_llvm_benchmark_path || '' }} \
             --mode="${MODE}" \
-            --benchmark=${{ matrix.type }}.json
+            --benchmark=${{ matrix.type }}${{ inputs.target-machine }}.json

```

Comment:
A delimiter needed?

Reply:
I was thinking about it. The thing is, the target machine can be empty, and then we'll have just an empty delimiter in the end. Adding additional logic with an optional `-` just for the internal name looked too much, so decided to go without a delimiter.

Reply:
`|| _default` maybe?
I'm thinking of making target mandatory.
Can it still be optional in this workflow?